src/README.md

```diff
@@ -151,6 +151,24 @@ is done executing. `Local` handles can only be allocated on the C++ stack.
 Most of the V8 API uses `Local` handles to work with JavaScript values or return
 them from functions.
 
+Additionally, according to [V8 public API documentation][`v8::Local<T>`], local handles
+(`v8::Local<T>`) should **never** be allocated on the heap.
+
+This disallows heap-allocated data structures containing instances of `v8::Local`
```

Comment:
Now come to think of it, when it's done we can probably add a rule to cpplint.py to just disallow `std::vector<Local>` or `std::vector<v8::Local>` (should be pretty trivial to implement).